en/docs/tutorials/integrating-with-solace.md

```diff
@@ -0,0 +1,144 @@
+# External Broker and Gateway Integration with API Manager
+
+WSO2 API Manager now supports integrating external third party Gateways and Brokers. Solace is an advanced event broker which is efficiently streaming events and information across cloud, on-premises, and IoT environments. In this section, you will integrate the Solace gateway environment with API Manager.
+
+## Integrating the Solace Event Broker
+
+1. Go to the [Solace PubHub+](https://solace.com/products/event-broker/) portal.
```

Comment:
To set up a solace PubHub+ account they have this Getting started guide. If its helpful and required we can add this here.

[1]. https://docs.solace.com/Cloud/ggs_signup.htm